packages/lib/config/rollup.config.js

```diff
@@ -0,0 +1,86 @@
+import typescript from 'rollup-plugin-typescript2';
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import json from '@rollup/plugin-json';
+import postcss from 'rollup-plugin-postcss';
+import replace from '@rollup/plugin-replace';
+import pkg from '../package.json';
+const currentVersion = require('./version')();
+
+if (process.env.CI !== 'true') {
+    console.warn(
+        '\x1b[33m%s\x1b[0m',
+        'Warning: Building custom bundle. We recommend using one of the official builds served by our servers or NPM. Check https://docs.adyen.com/checkout for more information.'
+    );
+}
+
+let cache;
```

Comment:
Any reason for this to be a `let` instead of a `const`?

Reply:
Not anymore! Good catch 👍